src/UI/MainMenuOptions.cpp

```diff
@@ -0,0 +1,370 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check it.
+// PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
+
+#include "MainMenuOptions.h"
+
+#include "NAS2D/Renderer/Renderer.h"
+
+#include "NAs2D/Utility.h"
```

Comment:
Casing typo. This will be an error on case sensitive filesystems, such as what Linux uses.